src/lib.rs

```diff
@@ -42,6 +42,7 @@ pub struct Analysis {
     pub refs: Vec<Ref>,
     pub macro_refs: Vec<MacroRef>,
     pub relations: Vec<Relation>,
+    pub borrows: Vec<Borrows>,
```

Comment:
OK, cool. Then maybe `per_fn_borrows` rather than just `borrows` for the field name? I think I prefer `BorrowData` rather than `Borrows`.